optaplanner-core/src/test/java/org/optaplanner/core/api/solver/SolverManagerTest.java

```diff
@@ -47,31 +57,35 @@
 
 public class SolverManagerTest {
 
-    @Test(timeout = 600_000)
+    @Test(timeout = 60_000)
     public void solveBatch_2InParallel() throws ExecutionException, InterruptedException {
-        CyclicBarrier barrier = new CyclicBarrier(2);
         final SolverConfig solverConfig = PlannerTestUtils.buildSolverConfig(TestdataSolution.class, TestdataEntity.class)
-                .withPhases(new CustomPhaseConfig().withCustomPhaseCommands(
-                        scoreDirector -> {
-                            try {
-                                barrier.await();
-                            } catch (InterruptedException | BrokenBarrierException e) {
-                                fail("Cyclic barrier failed.");
-                            }
-                        }), new ConstructionHeuristicPhaseConfig());
+                .withPhases(createPhaseWithConcurrentSolvingStart(2), new ConstructionHeuristicPhaseConfig());
         SolverManager<TestdataSolution, Long> solverManager = SolverManager.create(
                 solverConfig, new SolverManagerConfig().withParallelSolverCount("2"));
 
         SolverJob<TestdataSolution, Long> solverJob1 = solverManager.solve(1L,
-                PlannerTestUtils.generateTestdataSolution("s1"));
+                                                                           PlannerTestUtils.generateTestdataSolution("s1"));
         SolverJob<TestdataSolution, Long> solverJob2 = solverManager.solve(2L,
-                PlannerTestUtils.generateTestdataSolution("s2"));
+                                                                           PlannerTestUtils.generateTestdataSolution("s2"));
 
         assertSolutionInitialized(solverJob1.getFinalBestSolution());
         assertSolutionInitialized(solverJob2.getFinalBestSolution());
     }
 
-    @Test(timeout = 600_000)
+    private CustomPhaseConfig createPhaseWithConcurrentSolvingStart(int barrierPartiesCount) {
+        CyclicBarrier barrier = new CyclicBarrier(barrierPartiesCount);
+        return new CustomPhaseConfig().withCustomPhaseCommands(
+                scoreDirector -> {
+                    try {
+                        barrier.await();
+                    } catch (InterruptedException | BrokenBarrierException e) {
+                        fail("Cyclic barrier failed.");
+                    }
+                });
+    }
+
+    @Test(timeout = 60_000)
```

Comment:
Are you sure you want to bring the timeouts down to 1 minute isntead of 10 minutes? I remember we set it to 10 minutes for good reasons, even though the slowest test only took 10 seconds on our machines back then.

Reply:
Each test runs below 1 sec, I don't believe the CI environment will be > 60 times slower. If some test runs longer than 1 minute, I prefer to know about it (by seeing it failing on timeout).